test/unit/org/authInfoTest.ts

```diff
@@ -153,7 +153,6 @@ describe('AuthInfo', () => {
         // verify the returned object doesn't have secrets
         expect(() => walkAndSearchForSecrets(toJsonMap(fields) || {}, decryptedRefreshToken)).to.not.throw();
 
-        expect(strObj).does.not.include(ensureString(testOrg.defaultConnectedAppInfo.clientSecret));
```

Comment:
why don't we need these assertions anymore? Because clientSecret = `''`?

Reply:
Yep